source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2140,31 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, AIBridgeChannel_BridgeOffsetNullingCal_ReturnsError)
+{
+  const auto AI_MIN = 0.0;
+  const auto AI_MAX = 1.0;
+
+  EXPECT_THROW_DRIVER_ERROR({
+    auto status = create_ai_bridge_chan(AI_MIN, AI_MAX);
+  }, MEASUREMENT_TYPE_NOT_SUPPORTED_BY_SELECTED_PHYSICAL_CHANNEL_ERROR);
```

Comment:
This doesn't test PerformBridgeOffsetNullingCal. The precondition, "given AI bridge channel" fails.

As I suggested just now on https://github.com/ni/grpc-device/pull/1033#discussion_r1453469083

> When create channel fails, it leaves the task empty, so you can delete the top part of this test and get the same result.
>
> To test the bridge-(V/V)-specific API without a supported device, use either a task with the wrong channel type (e.g. strain or voltage) or an empty task.